docker-compose-tests.yml

```diff
@@ -14,6 +14,30 @@ services:
       - "8001:8000"
     working_dir: /home/dynamodblocal
 
+  cassandra:
+    image: cassandra:latest
+    networks:
+      - scylla
+    volumes:
+      - ./tests/docker/cassandra:/var/lib/cassandra
+    ports:
+      - "9043:9042"
+    expose:
+      - 9043
+
+  scylla-source:
+    image: scylladb/scylla:latest
+    networks:
+      - scylla
+    volumes:
+      - "./tests/docker/scylla-source:/var/lib/scylla"
+    ports:
+      - "8002:8000"
+      - "9044:9042"
+    expose:
+      - 9044
+    command: "--smp 1 --memory 2048M --alternator-port 8000 --alternator-write-isolation always_use_lwt"
```

Comment:
if you use policy
"only_rmw_uses_lwt" instead , tests will be faster and it is OK (safe) to use this mode since migrator doesn't require read before write afaik and since tests take 10mins maybe you can use it instead always

Reply:
Sure, I will try that! I don’t think Scylla’s is really the bottleneck in the CI. A lot of time is spent building some of the submodules too. If we get rid of our forks that will also help :)